Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'rake', '~> 13'
 gem 'redis', '~> 4'
 gem 'rotp', '~> 5'
 gem 'rqrcode'
-gem 'sass-rails', '~> 5'
+gem 'sass-rails', '~> 6.0'
```

Comment:
see https://github.com/alphagov/finder-frontend/issues/1124 and what we've done in finder-frontend
https://github.com/alphagov/finder-frontend/pull/1953